src/SWP/Bundle/CoreBundle/Resources/config/doctrine-orm/FacebookInstantArticlesArticle.orm.yml

```diff
@@ -0,0 +1,47 @@
+SWP\Bundle\CoreBundle\Model\FacebookInstantArticlesArticle:
+    type: entity
+    table: swp_fbia_article
+    id:
+        id:
+            type: integer
+            id: true
+            generator:
+                strategy: AUTO
+    fields:
+        tenantCode:
+            type: string
+        submissionId:
```

Comment:
mapping for fields specific to facebook IA should go to FBIABundle, here you would use just `tenantCode` and `article`